src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getFirstValue = a => {
```

Comment:
Not critical, but I would rather use more meaningful variable names here, like `firstNumber` etc.